ambry-clustermap/src/main/java/com.github.ambry.clustermap/FixedBackoffResourceStatePolicy.java

```diff
@@ -88,11 +89,10 @@ public boolean isDown() {
         down = true;
       }
     }
-    logger.trace("Resource " + resource + " is " + (down ? "down" : "not down")
-        + "; failureCount: " + failureCount.get()
-        + "; failureCountThreshold: " + failureCountThreshold
-        + "; remaining time: " + (downUntil.get() - SystemTime.getInstance().milliseconds())
-        + "; hard down: " + hardDown);
+    long remainingTime = downUntil.get() - SystemTime.getInstance().milliseconds();
+    logger.trace("Resource {} is {}; failureCount: {}; failureCountThreshold: {}; remaining time: {}; hard down: {}",
+        resource, (down ? "down" : "not down"), failureCount.get(), failureCountThreshold,
+        (remainingTime > 0 ? remainingTime : 0), hardDown);
```

Comment:
Because this can print a positive even when the resource is up and that could be misleading too. Would suggest printing this only if `resource` is up.